cicd/42c-conf-examples/README.md

```diff
@@ -12,6 +12,7 @@ This directory contains examples on each option for fine-tuning the CI/CD integr
 - `mapping-and-discovery`: Map API files to your existing APIs in 42Crunch Platform.
 - `mapping-no-discovery`: Switch off the discovery phase and only audit specifically mapped API files.
 - `branches-tags-and-prs`: Specify different configs for builds running on different branches/tags and PRs
+- `api_tags`: Specify the tags that will be added to all new apis (not existing in the collection before launch). The format is "tagCategory:tagName". All specified tags must already exist on the platform.
```

Comment:
Let's replace with: 'Tag APIs  with the tags defined in 42Crunch Platform. Only the newly created APIs are tagged. All specified tags must exist on the platform.'